### PR TITLE
Changed the comment

### PR DESCRIPTION
**Pull request by zhngx**
Changed the comment to direct the developers to the fabric react web page instead of the fabric core github repo.

### CODE REVIEW

File: src/taskpane/taskpane.html [hunk 1]
```diff
@@ -13,7 +13,7 @@
     <!-- Office JavaScript API -->
     <script type="text/javascript" src="https://appsforoffice.microsoft.com/lib/1.1/hosted/office.js"></script>
 
-    <!-- For the Office UI Fabric, go to http://aka.ms/office-ui-fabric to learn more. -->
+    <!-- For more information on Office UI Fabric, visit https://developer.microsoft.com/en-us/fabric. -->
```
Comment (akrantz) [inline src/taskpane/taskpane.html]: remove the `/en-us`

Comment (zhngx) [reply src/taskpane/taskpane.html]: /en-us is removed now

**Comment by akrantz** [comment]
@zhngx Most of these changes have already been merged in. Please rebase this off of the latest master branch, so there is only one commit which updates the comment. I can help in person if you're not familiar with this.

**Comment by akrantz** [comment]
Replaced with https://github.com/OfficeDev/Office-Addin-TaskPane/pull/21.

**Pull request closed without merging**